src/panels/lovelace/header-footer/hui-graph-header-footer.ts

```diff
@@ -100,10 +112,21 @@ export class HuiGraphHeaderFooter extends LitElement
 
   static get styles(): CSSResult {
     return css`
+      paper-spinner {
+        position: absolute;
+        top: calc(50% - 28px);
```

Comment:
You do realize this isn't the center of the container right? The center would be `calc(50% - 14px)`.

Reply:
This takes into account half of the spinner size, otherwise, you get this:
![Screenshot_20200415_170910](https://user-images.githubusercontent.com/28114703/79360478-e29d9e80-7f3b-11ea-9045-edd740fa7dea.png)